src/walletSdk/Anchor/index.ts

```diff
@@ -5,12 +5,15 @@ import { Auth } from "../auth";
 import { Interactive } from "../interactive";
 import { TomlInfo, parseToml } from "../toml";
 import { ServerRequestFailedError } from "../exception";
+import { Config } from "../";
 
 export class Anchor {
   private homeDomain = "";
+  private httpClient = null;
 
-  constructor(homeDomain) {
+  constructor(cfg: Config, homeDomain: string, httpClient) {
```

Comment:
I'm not an expert in javascript, if class being exported, does it mean constructor is exported too? 
I saw in js SDK all "internal" classes have comments to not create them directly. We should add some comments like that later to internal constructors too.

Reply:
Yes, the constructor will be exported, too

@Ifropc Can you please link one of the comments in js sdk you're referring to?

Reply:
https://github.com/stellar/js-stellar-sdk/blob/master/src/assets_call_builder.ts#L7

Reply:
Gotcha - was just making sure I understood. Yeah, this makes sense. We should include similar comments in our codebase

Reply:
👍  https://github.com/stellar/typescript-wallet-sdk/pull/5/commits/48b0bc4c95c87def43143a725f34ae15cc551b67